whitepapers/game-theory.md

```diff
@@ -185,7 +185,97 @@ For a full analysis of the tournament and its implications I recommend reading A
+
+- Stakers are rewarded with newly minted OHM, not whichever asset new entrants used to purchase their OHM. The protocol guarantees to back each OHM with 1 $DAI. Its ability to keep this promise is verifiable by anyone on-chain. The fact that OHM trades at a premium to 1 $DAI is a feature of market sentiment.  
+
+- In a ponzi there is no good or service being created, just money changing hands. Olympus has marketable services in the form of owned liquidity and bonding services for other protocols. As the treasury grows more opportunities will arise for Olympus to generate income.  
+
+- If large quantities of stakers suddenly decided to sell, they would first have to unstake. This raises APY for remaining stakers, limiting the volume of selling but if we imagine a doomsday scenario where everyone continued to sell, those last stakers would still get out with their initial investment and likely some profit (see [Asfi’s bankrun scenario](https://twitter.com/ishaheen10/status/1413369807450247168?s=21)).
+
+Bonding can also be thought of through the lens of reciprocity. 
+
+Bonders buy OHM from the protocol at a discount to the market price. This benefits the protocol by increasing its treasury or protocol owned liquidity (depending on the type of bond purchased). Naturally bonders can also become stakers once they have received their OHM.
+The reciprocal relationship is thus: I buy a bond so the protocol increases its treasury, and I receive discounted OHM in return. 
+
+A mutually beneficial exchange has been established between the Olympus protocol, bonders and stakers. 
+
+### Risks to Reciprocity in Olympus
+
+Stakers are incentivised by a high APY. Bonders are incentivised by discounted OHM. Can these incentives break? If so, how likely is it that the reciprocal relationship breaks down? 
+
+#### Stakers Reciprocity  
+
+First let's examine the necessary equations determining APY:
+
+$$
+APY = (1+rewardYield)1095
+$$
+
+Rebases occur 3 times a day, hence why the APY equation raises to 1095 (365 * 3 = 1095)
+
+$$
+rewardYield = OHMdistributed / OHMtotalStaked
+$$
+
+$$ 
+OHMdistributed = OHMtotalSupply * rewardRate
+$$
+
+$$
+rewardYield = (OHMtotalSupply * rewardRate) / OHMtotalStaked
+$$
+
+The reward rate is subject to change by community vote and can be thought of as the amount that OHMsupply increases per epoch that is then given to stakers. OHM total staked is decided by stakers. OHM total supply is a function of bond sales, staking rewards and minting for the DAO. These variables are exogenous as they are decided by forces outside of the model, so we should examine what happens to reward yield if any of them change and all other variables remain the same.  
+
+If $$ rewardRate $$ **falls** $$ rewardYield $$ **falls**. 
+If $$ OHMtotalStaked $$ **falls** $$ rewardYield $$ **rises**. 
+If $$ OHMtotalSupply $$ **rises** (but total staked stays constant i.e the percentage of staked OHM falls) $$ rewardYield $$ **rises**.
```

Comment:
```suggestion
- If $ rewardRate $ **falls**, $ rewardYield $ **falls**. 
- If $ OHM_{totalStaked} $ **falls**, $ rewardYield $ **rises**. 
- If $ OHM_{totalSupply} $ **rises**, $ rewardYield $ **rises**.
```

Reply:
We don't have to mention "but total staked stays constant..." since we have prefaced these scenarios by saying "while all other variables remain the same"